### PR TITLE
Granule storage aws-test branch

### PR DESCRIPTION
**Pull request by jennyhliu**
**Summary:** branch for testing granule-storage branch in aws

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## Test Plan
Things that should succeed before merging.

- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Update CHANGELOG
- [ ] Run `./bin/eslint-ratchet` and verify that eslint errors have not increased
  - [ ] Commit `.eslint-ratchet-high-water-mark` if the score has improved

## Release PR

If this is a **release** PR, make sure you branch name starts with `release-` prefix, e.g. `release-v-1.1.2`.

Reviewers: @tag-your-friends

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -102,6 +102,16 @@ jobs:
             if [[ "$CIRCLE_BRANCH" =~ release-.* ]] || [[ "$CIRCLE_BRANCH" =~ .*-aws-test ]] || [[  "$CIRCLE_BRANCH" == 'master' ]] || [[ ! -z "$RUN_INTEGRATION" ]]; then
               sudo chown -R circleci:circleci /usr/local/bin
 
+              # Build test processing task, CHANGE WHEN PUBLISHED TO NPM
```
Comment (kkelly51) [inline .circleci/config.yml]: So we'll be able to remove these as soon as it get's merged?

Comment (laurenfrederick) [reply .circleci/config.yml]: When the 1.6 release is published.

Comment (laurenfrederick) [reply .circleci/config.yml]: I'm planning on doing the release shortly after merging

File: packages/common/message-source.js [hunk 2]
```diff
@@ -68,6 +68,7 @@ class StateMachineS3MessageSource extends MessageSource {
     this.originalMessage = message;
 
     if (message.resources) {
+      // BUCKET/KEY TO DO - what bucket should this be?
```
Comment (kkelly51) [inline packages/common/message-source.js]: Found a TODO, What's the status up this? And the line under?

Comment (laurenfrederick) [reply packages/common/message-source.js]: Good catch. So it turned out `resources` is something we do not use anymore so we don't need to change it. There is some legacy code hanging around that is used by other teams. I'll remove the to do.

Comment (laurenfrederick) [reply packages/common/message-source.js]: This is code that has been replaced by the message adapter btw

**Pull request merged**